languages/go/exercises/concept/maps/.meta/example.go

```diff
@@ -0,0 +1,58 @@
+package maps
+
+// units store the Dozen Store unit measurement
+var units = map[string]int{
+	"quarter_of_a_dozen": 3,
+	"half_of_a_dozen":    6,
+	"dozen":              12,
+	"small_gross":        120,
+	"gross":              144,
+	"great_gross":        1728,
+}
+
+// NewBill create a new bill
+func NewBill() map[string]int {
+	return make(map[string]int)
+}
+
+// AddItem add item to customer bill
+func AddItem(bill map[string]int, item string, unit string) bool {
```

Comment:
I really like how you now pass the bill to each function!

Reply:
It's @tehsphinx idea not mine

Reply:
Well, then it is well-implemented :)